docs/contribute/debug-programs.md

```diff
@@ -1,50 +1,67 @@
-# Debugging with GDB
+# Debugging
 
-To debug the WebAssembly module, you can use GDB with Wasmtime. Ensure that you have compiled the module with the `-g` flag to include debugging information.
+## Debugging with GDB
 
-**NOTE**: currently this debugging tool does not support inspecting instructions. And operations like `layout split` and `si` might break the terminal. Using `layout src` is recommended.
+To debug a WebAssembly module using GDB, ensure that your module is compiled with debugging information (e.g., using the `-g` flag during compilation).
```

Comment:
Could there be a short example here of what a compilation with the flags you suggest would look like here?